app/models/product.rb

```diff
@@ -21,6 +21,14 @@ class Product < ApplicationRecord
     message: 'must be a URL for GIF, JPG, or PNG image.'
   }
 
+  after_initialize do |product|
+    product.title = 'abc' if product.title.blank?
```

Comment:
1. 'abc' should be a constant
2. Why you have used after_intialize here? What if just tries to use `Product.new` it will assign 'abc' as title but it should set title only while saving.